test/parallel/test-child-process-exec-detached.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+const notDetached = exec('git clone https://github.com/private/private', { timeout: 1000, detached: false });
```

Comment:
There is a good chance that `git` will be on the host system, but I don't think we should rely on `git`, and certainly not GitHub in our tests.